src/portal/src/app/project/p2p-provider/policy/policy.component.html

```diff
@@ -144,15 +144,17 @@ <h4 class="mt-1">
             <clr-dg-cell>{{p.description}}</clr-dg-cell>
         </clr-dg-row>
         <clr-dg-footer>
-            <span *ngIf="policyList?.length > 0">1 - {{policyList?.length}} {{'WEBHOOK.OF' | translate}} </span> {{policyList?.length}} {{'WEBHOOK.ITEMS' | translate}}
-            <clr-dg-pagination [clrDgPageSize]="10"></clr-dg-pagination>
+            <clr-dg-pagination [clrDgPageSize]="10">
+                <clr-dg-page-size [clrPageSizeOptions]="[10,25,50]">{{"PAGINATION.PAGE_SIZE" | translate}}</clr-dg-page-size>
```

Comment:
I notice in other page the default size is 15 and size opertions are [15,25,50]

Reply:
Default page size is 15. There is anather datagrid below this datagrid. So small size  is more suitable